api/src/utils/auth-utils.ts

```diff
@@ -1,161 +0,0 @@
-'use strict';
```

Comment:
Related to my other comment above, why are we are removing the authentication code?  We will need some kind of security around the api, so the public can't just call it.